src/Shared/LogMessagePacketBase.cs

```diff
@@ -205,6 +205,11 @@ internal enum LoggingEventType : int
         /// Event is <see cref="ExtendedCriticalBuildMessageEventArgs"/>
         /// </summary>
         ExtendedCriticalBuildMessageEvent = 33,
+
+        /// <summary>
+        /// Event is a ResponseGeneratedFileUsedEventArgs
+        /// </summary>
+        ResponseGeneratedFileUsedEvent = 34,
```

Comment:
This one should also probably be just `GeneratedFileUsedEventArgs`, hope I'm not misunderstanding.

Reply:
Done